client/my-sites/comments/comment-navigation/index.jsx

```diff
@@ -42,6 +43,13 @@ export class CommentNavigation extends Component {
 		status: 'unapproved',
 	};
 
+	bulkDeletePermanently = () => {
+		const { setBulkStatus, translate } = this.props;
+		if ( isUndefined( window ) || window.confirm( translate( 'Delete these comments permanently?' ) ) ) {
```

Comment:
Do we prefer using `window.confirm` for confirmation purposes instead of notices for example? Just asking because I don't remember seeing it in other parts of Calypso.